release_notes/45-release-notes.adoc

```diff
@@ -0,0 +1,421 @@
+
+[id="enhanced-acscs-experience_{context}"]
+=== Enhanced {product-title-managed-short} experience
+
+We have improved the {product-title-managed-short} experience on `console.redhat.com`, making signing up for a trial and getting support easier.
+
+[id="notable-technical-changes_{context}"]
+== Notable technical changes
+
+* For {product-title-short} Operator, Red{nbsp}Hat has added a label selector for caching configuration of secrets and configmaps, resulting in a significant reduction in memory consumption, especially on large clusters. In testing, we have seen a 28% decrease in memory usage on a new {ocp} cluster.
+
+* The {product-title-short} Operator now adds a label `app.stackrox.io/managed-by: operator` to all Helm chart resources and secrets created by the operator, providing better organization and visibility.
+
+* The {product-title-short} Operator increases the number of requests sent to the API server to retrieve secrets not managed by the Operator and do not match the cache label selector.
+
+* Scanner DB now runs on PostgreSQL 15, replacing the earlier version, PostgreSQL 12. Since the database is not persisted, no migration is required, and you can continue using the scanner without any additional steps.
+
+* The Nexus and Red{nbsp}Hat registry integrations now attempt to pull manifest digests using a HEAD request to `/v2/<name>/manifests/<reference>`. This change resolves an issue that previously resulted in an `unsupported digest algorithm error` when using Scanner V4. You can turn off this new behavior by setting the environment variable `ROX_ATTEMPT_MANIFEST_DIGEST` to `false`.
+
+* {product-title-short} includes new policy categories, and some default policies are tagged with these new policy categories.
++
+.New policy categories
+[options="header"]
+|====
+
+|Policy category|Policy
+
+.2+|*Zero Trust*
+|Deployments should have at least one ingress Network Policy
+|Unauthorized Network Flow
+
+.7+|*Supply Chain Security*
+|Images with no scans
+|30-day Scan Age
+|90-day Image Age
+|Required Annotation: Email
+|Required Annotation: Owner/Team
+|Required Label: Owner/Team
+|Latest tag
+
+|====
+
+* When generating manifests for {ocp}, `roxctl` now defaults to {ocp} 4.x instead of {ocp} 3.x.
+
+* You can now delegate the Image scans triggered by image watch reprocessing based on the delegated scanning configuration. To turn off this feature, set the Central environment variable `ROX_DELEGATE_WATCHED_IMAGE_REPROCESSING` to `false`.
+
+* The Scanner V4 Matcher now performs concurrent vulnerability updates using iterators, reducing memory consumption from 4GB to 500MB.
+
+* {product-title-short} 4.5 lazily populates the initial registry integration repository list `/v2/_catalog` to improve Central startup performance. This change reduces startup time in environments with many autogenerated integrations.
```

Comment:
```suggestion
* {product-title-short} 4.5 slowly populates the initial registry integration repository list `/v2/_catalog` to improve Central startup performance. This change reduces startup time in environments with many autogenerated integrations.
```
I'd use "slowly" or another synonym here, unless "lazily" is the accepted technical usage.